shopinvader_product_variant_selector/tests/test_shopinvader_variant.py

```diff
@@ -0,0 +1,335 @@
+# Copyright 2019 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.tests.common import SavepointCase
+
+
+class ShopinvaderVariantCase(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        ref = cls.env.ref
+        cls.backend = ref("shopinvader.backend_1")
+        cls.lang = ref("base.lang_en")
+        cls.template = cls.mref("product_template_1")
+        cls.maxDiff = None
+
+    @classmethod
+    def mref(cls, key):
```

Comment:
`mref` stands for?

Reply:
"module ref" it avoid me to write env.ref('shopinvader_product_variant_selector.**my-key**') => mref('**my-key**')